go/database/mpt/io/archive.go

```diff
@@ -267,6 +267,7 @@ func importArchive(liveDbDir, archiveDbDir string, in io.Reader) (err error) {
 	}()
 
 	// Restore the archive from the input file.
+	progress := logger.NewProgressTracker("processed %d blocks, rate: %f blocks/s", 1_000_000)
```

Comment:
nit: progress -> tracker?
Also isnt 1m blocks a bit too shallow? I think 100k is just enough

Reply:
when importing current blockchain it prints about 80 lines, if the window was set to 100k, it would print 800 lines. What is better, I do not know ...

Reply:
I guess it does not really matter. I will leave the decision on you.